### ISSUE TITLE
implement loss of remanence due to warming through Verwey transition in MPMS_analysis.ipynb

### ISSUE THREAD

**Issue by Swanson-Hysell** [PmagPy/RockmagPy-notebooks#2]
Jackson and Moskowitz (2020) describe that: *The IRMDB software allows interactive analysis and calculation of TV, as well as the loss of remanence due to warming through the transition, using the derivative method of Liu et al. (2004)*. This loos of remanence calculation should be implemented in MPMS_analysis.ipynb.

**Comment by Swanson-Hysell** [comment]
Hey @peat22, can you have a look at how this is implemented in the Delphi/Pascal in Quantum Ferret? I am thinking that it makes sense to use the "magnetite curve" determined from the background fit. It seems one could either use the start and end of the magnetite curve (i.e. the first and last blue point blue). Or alternatively could take the excluded temperature range start and end as the before and after remanence values (either approach should give about the same answer if the background is decently fit). 

<img width="505" alt="Screenshot 2024-02-12 at 10 35 31 AM" src="https://github.com/PmagPy/RockmagPy-notebooks/assets/4332322/b390ee56-4e67-4a06-900f-aa09cabc80f4">

**Comment by peat22** [comment]
I'll take a look at this tomorrow. Had a fun day struggling to get anaconda
installed and working (Thanks Plinio!). And I thought Delphi was difficult
to install!

**Comment by Swanson-Hysell** [comment]
I have added initial functionality to do this calculation to the function `verwey_estimate()` with this code in commit 69ac714

```
remanence_loss = mgt_curve.iloc[0] - mgt_curve.iloc[-1]
```

and the notebook is updated as well to ingest these values and tabulate them 3ae66a10b5c60f61cc1643da94e7308f6f90f0d7. The next step is to compare to values that are determined within the IRM Quantum Ferret software and to figure out what the relative value is within that software as well.

**Comment by Swanson-Hysell** [comment]
This first and last magnetite point approach gives slightly different values. Here are the current values calculated in the notebook:

| specimen  | verwey_temp | remanence_loss | t_range_background_min | t_range_background_max | excluded_t_min | excluded_t_max | poly_deg |
|-----------|-------------|----------------|------------------------|------------------------|----------------|----------------|----------|
| NED18-2c  | 105.861068  | 0.003426       | 60                     | 250                    | 75             | 150            | 3        |
| NED2-8c   | 116.140403  | 0.001674       | 50                     | 260                    | 75             | 150            | 3        |
| NED4-1c   | 117.902959  | 0.000448       | 60                     | 260                    | 75             | 150            | 3        |

Here are the values that come out of TVviewer:

| specimen  | verwey_temp | remanence_loss | t_range_background_min | t_range_background_max | excluded_t_min | excluded_t_max | poly_deg |
|-----------|-------------|----------------|------------------------|------------------------|----------------|----------------|----------|
| NED18-2c  | 1059  | 0.00318       | 60                     | 250                    | 75             | 150            | 3        |
| NED2-8c   | 116.2  | 0.00138       | 50                     | 260                    | 75             | 150            | 3        |
| NED4-1c   | 117.9  | 0.000440       | 60                     | 260                    | 75             | 150            | 3        |

The Verwey temperature estimates look to be right on, while the delta FC [Am2/kg] are slightly different.

**Comment by Swanson-Hysell** [comment]
Rather than 

```
remanence_loss = mgt_curve.iloc[0] - mgt_curve.iloc[-1]
```

It makes sense to integrate the magnetite derivative curve which can be implemented with:

```
remanence_loss = np.trapz(mgt_dM_dT, temps_dM_dT_background)
```

**Comment by Swanson-Hysell** [comment]
The resulting values for the example ECMB data are below and are closer to the values out of TVviewer:

| specimen | verwey_temp | remanence_loss | t_range_background_min | t_range_background_max | excluded_t_min | excluded_t_max | poly_deg |
|----------|-------------|----------------|------------------------|------------------------|----------------|----------------|----------|
| NED18-2c | 105.861068  | 0.003278       | 60                     | 250                    | 75             | 150            | 3        |
| NED2-8c  | 116.145701  | 0.001548       | 60                     | 260                    | 75             | 150            | 3        |
| NED4-1c  | 117.902959  | 0.000437       | 60                     | 260                    | 75             | 150            | 3        |
